.gitignore

```diff
@@ -8,4 +8,7 @@ build
 .idea
 
 #OSX
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Claude local configuration
+CLAUDE.local.md
```

Comment:
the use of a local file like this seems deprecated but I guess there might be leftovers. It does not hurt having this line here